paymentsource/client.go

```diff
@@ -150,16 +150,20 @@ func Verify(id string, params *stripe.SourceVerifyParams) (*stripe.PaymentSource
 
 func (s Client) Verify(id string, params *stripe.SourceVerifyParams) (*stripe.PaymentSource, error) {
 	body := &stripe.RequestValues{}
-	body.Add("amounts[]", strconv.Itoa(int(params.Amounts[0])))
-	body.Add("amounts[]", strconv.Itoa(int(params.Amounts[1])))
 
 	source := &stripe.PaymentSource{}
 	var err error
 
 	if len(params.Customer) > 0 {
+		body.Add("amounts[]", strconv.Itoa(int(params.Amounts[0])))
+		body.Add("amounts[]", strconv.Itoa(int(params.Amounts[1])))
+
 		err = s.B.Call("POST", fmt.Sprintf("/customers/%v/sources/%v/verify", params.Customer, id), s.Key, body, &params.Params, source)
 	} else {
-		err = errors.New("Only customer bank accounts can be verified in this manner.")
+		body.Add("values[]", strconv.Itoa(int(params.Amounts[0])))
```

Comment:
@stan-stripe You have lots of context on this, so would you mind pitching an alternative? Should we keep `Amounts` for customer-specific verification and add a `Values` here? Should we keep the two-element array?